ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -304,10 +304,14 @@
       + "KSQL metastore backup files are located.";
 
   public static final String KSQL_SUPPRESS_ENABLED = "ksql.suppress.enabled";
-  public static final Boolean KSQL_SUPPRESS_ENABLED_DEFAULT = false;
+  public static final Boolean KSQL_SUPPRESS_ENABLED_DEFAULT = true;
   public static final String KSQL_SUPPRESS_ENABLED_DOC =
       "Feature flag for suppression, specifically EMIT FINAL";
 
+  public static final String KSQL_SUPPRESS_BUFFER_SIZE = "ksql.suppress.buffer.size";
+  public static final Long KSQL_SUPPRESS_BUFFER_SIZE_DEFAULT = -1L;
+  public static final String KSQL_SUPPRESS_BUFFER_SIZE_DOC =
+      "Bound the size of the buffer used for suppression";
```

Comment:
let's explicitly call out that `-1L` is "unbounded" and also we should describe the failure behavior (i.e. what happens if the bound is filled?)